public/index.html

```diff
@@ -45,6 +45,62 @@ <h3 class="oxford location-heading">Oxford</h3>
             </div>
           </div>
 
+          <div class="job">
+            <p><a class="job-title">Paid Web Developer Internship</a></p>
+            <div class="job-description">
+              <p>
+                We are excited to offer a paid 3-month, web developer internship at the Zooniverse.
+                The candidate would ideally be able to travel to our office in Oxford (UK), however
+                we will also consider candidates who wish to work remotely.
+
+                The internship will run over summer 2017 with the option for flexible start and end dates.
+              </p>
+              <p>
+                As an intern you will work on a variety of projects, according to our current needs
+                as well as the interests and skills of the successful candidate.
```

Comment:
I would change this to

> as well as your own skills and interests.

Since the sentence is addressing the candidate themselves.